src/note.cpp

```diff
@@ -467,6 +467,10 @@ void Note::UpdateFromTransPitch(TransPitch tp)
     Accid *accid = this->GetDrawingAccid();
     bool transposeGesturalAccid = false;
     bool transposeWrittenAccid = false;
+    if (!accid) {
```

Comment:
We probably want to add it only if necessary, don't we? So maybe first see boolean values only if `accid` exists, and then add it if necessary below. Would this work?

Reply:
From https://github.com/rism-ch/verovio/pull/1225#discussion_r355787424, we talked about what to do if there was no accidental given, and decided to always include the `@accid.ges="n"` in transposed scores. (So, we're always going to need `accid`).

Reply:
(With the current implementation, the only cases where this will add an empty `<accid>` object is with microtones and invalid transpositions)

Reply:
Yes, the rule should be (1) never create `note@accid` (or equivalent in a child accid element), (2) never create `note@accid.ges` if `note@accid` is present, and  (3) only create `note@accid.ges` when there is an implicit `note@accid.ges="n"` and the implicit `n` changes to some other accidental (in which case the `note@accid.ges` value needs to become explicit).

Reply:
Ready to merge, then?